File: M3/Practica6/Exercici1.java
import java.io.*; 
public class Exercici1 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Introduce texto=");
        String texto = reader.readLine();
        System.out.println("Introduce un caracter=");
        String caracter = reader.readLine();
        if (texto.indexOf(caracter) > -1){
			System.out.print("El caracter '"+caracter+"' aparece en '"+texto+"'");
		}
		else{
			System.out.print("El caracter '"+caracter+"' no aparece en '"+texto+"'");
		}
			
    }
}

File: M3/Practica6/Exercici1.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
texto=raw_input("Introduce texto=")
caracter=raw_input("Introduce un caracter=")
if caracter in texto:
	print "El caracter",caracter,"aparece en",texto
else:
	print "El caracter",caracter,"no aparece en",texto
